src/services/upload-manager.ts

```diff
@@ -79,8 +99,64 @@ export class UploadManager extends EventEmitter {
     }
   }
 
+  private async uploadGroup(creationDetails: UserCreationDetails, places: Place[], api: ChtApi) {
+    if (!creationDetails.username || !creationDetails.placeId) {
+      throw new Error('creationDetails must not be empty');
+    }
+    const placeIds: {[key:string]: any} = { [creationDetails.placeId]: '' };
+    for (const place of places) {
+      if (place.creationDetails.placeId) {
+        placeIds[place.id] = undefined;
+        continue;
+      }
+      this.eventedPlaceStateChange(place, PlaceUploadState.IN_PROGRESS);
+
+      try {
+        const payload = place.asChtPayload(api.chtSession.username, creationDetails.contactId);
+        await Config.mutate(payload, api, place.isReplacement);
```

Comment:
i'm surprised this can't be implemented with more reuse. is a variation of `uploadSinglePlace` not possible?

Reply:
There's not reuse of this function, it's only used once here